docs/content/快速上手/mysql-starrocks-pipeline-tutorial-zh.md

```diff
@@ -0,0 +1,284 @@
+# 基于 Flink CDC 3.0 构建 MySQL 到 Starrocks 的 Streaming ELT
```

Comment:
```suggestion
# 基于 Flink CDC 3.0 构建 MySQL 到 StarRocks 的 Streaming ELT
```

Reply:
StarRocks is preferred. Please help to replace all Starrocks to StarRocks in the doc. Thanks.